blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/UserSearch.java

```diff
@@ -23,9 +26,27 @@ public String getType() {
     @Override
     public Pageable<BlueUser> search(Query q) {
         List<BlueUser> users = new ArrayList<>();
+        BlueOrganization organization = getOrganization(q);
```

Comment:
I think the `getDefault()` suggestion would solve this, as `OrganizatinFactory` implementors could contribute a specific default organization